storyblok/storyblok-design-system
# feat(icon): add play-circle icon

Introduces the `play-circle` icon

## Pull request type

Jira Link: [WORK-168](https://storyblok.atlassian.net/browse/WORK-168)




## How to test this PR

Run storybook and make sure a nem icon named `play-circle` is being displayed properly in the `Components > SbIcon > All Icons` section


## What is the new behavior?



- A new icon named `play-circle` must be available for use

## Other information
